resources/js/components/layout/NavSearch.vue

```diff
@@ -273,6 +277,10 @@ export default {
         showBookmarks() {
             this.show_bookmarks = true;
         },
+        searchFulltext() {
+            console.log(this.texts.fulltext_route);
+            window.location.href = `${this.texts.fulltext_route}?term=${this.term}`;
```

Comment:
Please make it a proper link to be accessible to e-readers
```
Why not just make it a link ```
<a
            v-bind:href="searchFullTextUrl()"
```
or something :)